doc/nrf/device_guides/working_with_nrf/nrf70/nrf7002ek_gs.rst

```diff
@@ -0,0 +1,169 @@
+.. _ug_nrf7002ek_gs:
+
+Getting started with nRF7002 EK
+###############################
+
+.. contents::
+   :local:
+   :depth: 4
+
+This section gets you started with your nRF7002 :term:`Evaluation Kit (EK)` using the |NCS|.
+
+The nRF7002 EK is a versatile evaluation kit in the form of an Arduino shield.
+The kit can be used to provide Wi-Fi connectivity and Wi-Fi-based locationing to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion IC.
+
+The nRF7002 EK features the nRF7002 companion IC.
+In addition, the shield may be used to emulate the nRF7001 and nRF7000 companion IC variants.
+
+If this is your first time developing with a Nordic DK, read the appropriate getting started tutorial first:
+
+* :ref:`ug_nrf7002_gs`
+* :ref:`ug_nrf52_gs`
+* :ref:`ug_nrf5340_gs`
+* :ref:`ug_nrf9160_gs`
+
+Overview
+********
+
+The nRF7002 EK (PCA63556) is designed to provide Wi-Fi connectivity and Wi-Fi (SSID) scanning capabilities through the nRF7002 companion IC to a compatible host development board.
+
+The nRF7002 EK features an Arduino shield form factor and interface connector that allows it to be used with Arduino compatible boards, such as the `nRF52840 DK <nRF52840 DK product page_>`_, `nRF5340 DK <nRF5340 DK product page_>`_, `nRF9160 DK <nRF9160 DK product page_>`_, or `nRF9161 DK <Nordic nRF9161 DK_>`_.
```

Comment:
You list 9161 here, but 9161 is missing from the Supported Boards' table in the other section.

Reply:
If it is about the Features page, then it will be handled in https://nordicsemi.atlassian.net/browse/NCSDK-22161.

Reply:
Thanks